### PR TITLE
Use typeshed.Incomplete and object instead of Any in tkinter stubs

### PR DESCRIPTION
**Pull request by Akuli**
Created by replacing all `Any`s with `Incomplete` and then reviewing manually with `git add -p`.

Tkinter had quite a lot of `Any` for a few reasons:
- Old versions of stubgen generated `Any | None` (actually `Optional[Any]`) for parameters with a `None` default value.
- I used `Any` in callables before I learned about `Callable[[foo], object]` for ignored return values.
- Things that are impossible to type accurately. I left these as `Any`.

### CODE REVIEW

File: stdlib/tkinter/__init__.pyi [hunk 1]
```diff
@@ -490,7 +492,7 @@ class Misc:
     def _windowingsystem(self) -> Literal["win32", "aqua", "x11"]: ...
     def nametowidget(self, name: str | Misc | _tkinter.Tcl_Obj) -> Any: ...
     def register(
-        self, func: Callable[..., Any], subst: Callable[..., Sequence[Any]] | None = ..., needcleanup: int = ...
+        self, func: Callable[..., Incomplete], subst: Callable[..., Sequence[Incomplete]] | None = ..., needcleanup: int = ...
```
Comment (Akuli) [inline stdlib/tkinter/__init__.pyi:495]:
```suggestion
        self, func: Callable[..., object], subst: Callable[..., Sequence[Any]] | None = ..., needcleanup: int = ...
```

File: stdlib/tkinter/__init__.pyi [hunk 2]
```diff
@@ -581,24 +583,24 @@ class Misc:
...
     def image_types(self) -> tuple[str, ...]: ...
     # See #4363 and #4891
-    def __setitem__(self, key: str, value: Any) -> None: ...
-    def __getitem__(self, key: str) -> Any: ...
-    def cget(self, key: str) -> Any: ...
-    def configure(self, cnf: Any = ...) -> Any: ...
+    def __setitem__(self, key: str, value: Incomplete) -> None: ...
+    def __getitem__(self, key: str) -> Incomplete: ...
+    def cget(self, key: str) -> Incomplete: ...
+    def configure(self, cnf: Incomplete = ...) -> Incomplete: ...
```
Comment (Akuli) [inline stdlib/tkinter/__init__.pyi:589]:
```suggestion
    def __setitem__(self, key: str, value: Any) -> None: ...
    def __getitem__(self, key: str) -> Any: ...
    def cget(self, key: str) -> Any: ...
    def configure(self, cnf: Any = ...) -> Any: ...
```

File: stdlib/tkinter/__init__.pyi [hunk 3]
```diff
@@ -625,11 +627,11 @@ class Wm:
     ) -> tuple[int, int, int, int] | None: ...
     aspect = wm_aspect
     @overload
-    def wm_attributes(self) -> tuple[Any, ...]: ...
+    def wm_attributes(self) -> tuple[Incomplete, ...]: ...
```
Comment (Akuli) [inline stdlib/tkinter/__init__.pyi:630]:
```suggestion
    def wm_attributes(self) -> tuple[Any, ...]: ...
```

File: stdlib/tkinter/__init__.pyi [hunk 4]
```diff
@@ -625,11 +627,11 @@ class Wm:
...
     @overload
-    def wm_attributes(self, __option: str) -> Any: ...
+    def wm_attributes(self, __option: str) -> Incomplete: ...
     @overload
-    def wm_attributes(self, __option: str, __value: Any, *__other_option_value_pairs: Any) -> None: ...
+    def wm_attributes(self, __option: str, __value: Incomplete, *__other_option_value_pairs: Incomplete) -> None: ...
```
Comment (Akuli) [inline stdlib/tkinter/__init__.pyi:634]:
```suggestion
    def wm_attributes(self, __option: str, __value: Incomplete, *__other_option_value_pairs: Any) -> None: ...
```

File: stdlib/tkinter/commondialog.pyi [hunk 5]
```diff
@@ -1,13 +1,14 @@
...
 class Dialog:
     command: ClassVar[str | None]
-    master: Any | None
-    options: Mapping[str, Any]
-    def __init__(self, master: Any | None = ..., **options) -> None: ...
-    def show(self, **options) -> Any: ...
+    master: Incomplete | None
+    options: Mapping[str, Incomplete]
+    def __init__(self, master: Incomplete | None = ..., **options) -> None: ...
+    def show(self, **options) -> Incomplete: ...
```
Comment (Akuli) [inline stdlib/tkinter/commondialog.pyi:14]:
```suggestion
    def show(self, **options): ...
```

File: stdlib/tkinter/tix.pyi [hunk 6]
```diff
@@ -36,12 +37,12 @@ TCL_ALL_EVENTS: Literal[0]
 
 class tixCommand:
     def tix_addbitmapdir(self, directory: str) -> None: ...
-    def tix_cget(self, option: str) -> Any: ...
-    def tix_configure(self, cnf: dict[str, Any] | None = ..., **kw: Any) -> Any: ...
+    def tix_cget(self, option: str) -> Incomplete: ...
+    def tix_configure(self, cnf: dict[str, Any] | None = ..., **kw) -> Incomplete: ...
```
Comment (Akuli) [inline stdlib/tkinter/tix.pyi:41]:
```suggestion
    def tix_cget(self, option: str) -> Any: ...
    def tix_configure(self, cnf: dict[str, Any] | None = ..., **kw: Any) -> Any: ...
```

File: stdlib/tkinter/tix.pyi [hunk 7]
```diff
@@ -36,12 +37,12 @@ TCL_ALL_EVENTS: Literal[0]
...
     def tix_filedialog(self, dlgclass: str | None = ...) -> str: ...
     def tix_getbitmap(self, name: str) -> str: ...
     def tix_getimage(self, name: str) -> str: ...
-    def tix_option_get(self, name: str) -> Any: ...
+    def tix_option_get(self, name: str): ...
```
Comment (Akuli) [inline stdlib/tkinter/tix.pyi:45]:
```suggestion
    def tix_option_get(self, name: str) -> Any: ...
```

File: stdlib/tkinter/tix.pyi [hunk 8]
```diff
@@ -53,15 +54,15 @@ class TixWidget(tkinter.Widget):
         master: tkinter.Misc | None = ...,
         widgetName: str | None = ...,
         static_options: list[str] | None = ...,
-        cnf: dict[str, Any] = ...,
-        kw: dict[str, Any] = ...,
+        cnf: dict[str, Incomplete] = ...,
+        kw: dict[str, Incomplete] = ...,
```
Comment (Akuli) [inline stdlib/tkinter/tix.pyi:58]:
```suggestion
        cnf: dict[str, Any] = ...,
        kw: dict[str, Any] = ...,
```

File: stdlib/tkinter/tix.pyi [hunk 9]
```diff
@@ -53,15 +54,15 @@ class TixWidget(tkinter.Widget):
...
     ) -> None: ...
-    def __getattr__(self, name: str) -> Any: ...
+    def __getattr__(self, name: str) -> Incomplete: ...
     def set_silent(self, value: str) -> None: ...
     def subwidget(self, name: str) -> tkinter.Widget: ...
     def subwidgets_all(self) -> list[tkinter.Widget]: ...
-    def config_all(self, option: Any, value: Any) -> None: ...
-    def image_create(self, imgtype: str, cnf: dict[str, Any] = ..., master: tkinter.Widget | None = ..., **kw: Any) -> None: ...
+    def config_all(self, option: Incomplete, value: Incomplete) -> None: ...
```
Comment (Akuli) [inline stdlib/tkinter/tix.pyi:64]:
```suggestion
    def config_all(self, option: Any, value: Any) -> None: ...
```

File: stdlib/tkinter/tix.pyi [hunk 10]
```diff
@@ -70,102 +71,102 @@ class TixSubWidget(TixWidget):
     ) -> None: ...
 
 class DisplayStyle:
-    def __init__(self, itemtype: str, cnf: dict[str, Any] = ..., *, master: tkinter.Widget | None = ..., **kw: Any) -> None: ...
-    def __getitem__(self, key: str) -> Any: ...
-    def __setitem__(self, key: str, value: Any) -> None: ...
+    def __init__(self, itemtype: str, cnf: dict[str, Incomplete] = ..., *, master: tkinter.Widget | None = ..., **kw) -> None: ...
+    def __getitem__(self, key: str) -> Incomplete: ...
+    def __setitem__(self, key: str, value: Incomplete) -> None: ...
```
Comment (Akuli) [inline stdlib/tkinter/tix.pyi:76]:
```suggestion
    def __setitem__(self, key: str, value: Any) -> None: ...
```

File: stdlib/tkinter/tix.pyi [hunk 11]
```diff
@@ -178,24 +179,24 @@ class HList(TixWidget, tkinter.XView, tkinter.YView):
...
+    def indicator_configure(self, entry: str, cnf: dict[str, Incomplete] = ..., **kw) -> Incomplete | None: ...
+    def indicator_cget(self, entry: str, opt: Incomplete) -> Incomplete: ...
     def indicator_exists(self, entry: str) -> bool: ...
     def indicator_delete(self, entry: str) -> None: ...
     def indicator_size(self, entry: str) -> int: ...
     def info_anchor(self) -> str: ...
     def info_bbox(self, entry: str) -> tuple[int, int, int, int]: ...
     def info_children(self, entry: str | None = ...) -> tuple[str, ...]: ...
-    def info_data(self, entry: str) -> Any: ...
+    def info_data(self, entry: str) -> Incomplete: ...
```
Comment (Akuli) [inline stdlib/tkinter/tix.pyi:199]:
```suggestion
    def info_data(self, entry: str) -> Any: ...
```

File: stdlib/tkinter/tix.pyi [hunk 12]
```diff
@@ -258,44 +259,44 @@ class TList(TixWidget, tkinter.XView, tkinter.YView):
...
 class PanedWindow(TixWidget):
-    def __init__(self, master: tkinter.Widget | None, cnf: dict[str, Any] = ..., **kw: Any) -> None: ...
-    def add(self, name: str, cnf: dict[str, Any] = ..., **kw: Any) -> None: ...
+    def __init__(self, master: tkinter.Widget | None, cnf: dict[str, Incomplete] = ..., **kw) -> None: ...
+    def add(self, name: str, cnf: dict[str, Incomplete] = ..., **kw) -> None: ...
     def delete(self, name: str) -> None: ...
     def forget(self, name: str) -> None: ...  # type: ignore[override]
-    def panecget(self, entry: str, opt: Any) -> Any: ...
-    def paneconfigure(self, entry: str, cnf: dict[str, Any] = ..., **kw: Any) -> Any | None: ...
+    def panecget(self, entry: str, opt: Incomplete) -> Incomplete: ...
```
Comment (Akuli) [inline stdlib/tkinter/tix.pyi:271]:
```suggestion
    def panecget(self, entry: str, opt: Incomplete): ...
```

File: stdlib/tkinter/tix.pyi [hunk 13]
```diff
@@ -258,44 +259,44 @@ class TList(TixWidget, tkinter.XView, tkinter.YView):
...
 
 class InputOnly(TixWidget):
-    def __init__(self, master: tkinter.Widget | None = ..., cnf: dict[str, Any] = ..., **kw: Any) -> None: ...
+    def __init__(self, master: tkinter.Widget | None = ..., cnf: dict[str, Incomplete] = ..., **kw) -> None: ...
 
 class Form:
-    def __setitem__(self, key: str, value: Any) -> None: ...
-    def config(self, cnf: dict[str, Any] = ..., **kw: Any) -> None: ...
-    def form(self, cnf: dict[str, Any] = ..., **kw: Any) -> None: ...
+    def __setitem__(self, key: str, value: Incomplete) -> None: ...
```
Comment (Akuli) [inline stdlib/tkinter/tix.pyi:295]:
```suggestion
    def __setitem__(self, key: str, value: Any) -> None: ...
```

File: stdlib/tkinter/ttk.pyi [hunk 14]
```diff
@@ -1137,15 +1138,15 @@ class Treeview(Widget, tkinter.XView, tkinter.YView):
         background: tkinter._Color = ...,
         font: _FontDescription = ...,
         image: tkinter._ImageSpec = ...,
-    ) -> _TreeviewTagDict | Any: ...  # can be None but annoying to check
+    ) -> _TreeviewTagDict | Incomplete: ...  # can be None but annoying to check
```
Comment (Akuli) [inline stdlib/tkinter/ttk.pyi:1141]:
```suggestion
    ) -> _TreeviewTagDict | Any: ...  # can be None but annoying to check
```

File: stdlib/tkinter/__init__.pyi [hunk 15]
```diff
@@ -277,11 +279,11 @@ def NoDefaultRoot() -> None: ...
 _TraceMode: TypeAlias = Literal["array", "read", "write", "unset"]
 
 class Variable:
-    def __init__(self, master: Misc | None = ..., value: Any | None = ..., name: str | None = ...) -> None: ...
-    def set(self, value: Any) -> None: ...
+    def __init__(self, master: Misc | None = ..., value: Incomplete | None = ..., name: str | None = ...) -> None: ...
```
Comment (srittau) [inline stdlib/tkinter/__init__.pyi:282]: Cases like this should just be kept unannotated. `Incomplete` is intended for partially unknown types, which we can't just leave unannotated.

Comment (AlexWaygood) [reply stdlib/tkinter/__init__.pyi:282]: I was going to say that `Any | None` is more type-safe than `Any`, but I guess it doesn't actually give you any more type safety for argument types, only for attribute annotations and return types.

File: stdlib/tkinter/__init__.pyi [hunk 16]
```diff
@@ -195,7 +195,9 @@ _Padding: TypeAlias = Union[
 ]
 _Relief: TypeAlias = Literal["raised", "sunken", "flat", "ridge", "solid", "groove"]  # manual page: Tk_GetRelief
 _ScreenUnits: TypeAlias = str | float  # Often the right type instead of int. Manual page: Tk_GetPixels
-_XYScrollCommand: TypeAlias = str | Callable[[float, float], Any]  # -xscrollcommand and -yscrollcommand in 'options' manual page
+_XYScrollCommand: TypeAlias = (
+    str | Callable[[float, float], object]
+)  # -xscrollcommand and -yscrollcommand in 'options' manual page
```
Comment (AlexWaygood) [inline stdlib/tkinter/__init__.pyi:200]: black screwed up the formatting here

```suggestion
# -xscrollcommand and -yscrollcommand in 'options' manual page
_XYScrollCommand: TypeAlias = str | Callable[[float, float], object]
```

File: stdlib/tkinter/__init__.pyi (hunk 15 above)
Comment (srittau) [reply stdlib/tkinter/__init__.pyi:282]: It still makes sense to use `Incomplete | None` in argument positions as it's one more step towards correctness and at least serves a documentarion.

Comment (AlexWaygood) [reply stdlib/tkinter/__init__.pyi:282]: > It still makes sense to use `Incomplete | None` in argument positions as it's one more step towards correctness and at least serves a documentarion.

Oh, I thought that was what you were arguing against here. Which specific change are you commenting on? :)

Comment (srittau) [reply stdlib/tkinter/__init__.pyi:282]: Oops, seems I misclicked the line. That happens when doing stuff like this on the phone. I mean cases like `set` and `get` below.

Comment (AlexWaygood) [reply stdlib/tkinter/__init__.pyi:282]: Ignore everything I said in response to your comment in that case; we're in agreement 😄

File: stdlib/tkinter/__init__.pyi [hunk 17]
```diff
@@ -277,11 +278,11 @@ def NoDefaultRoot() -> None: ...
 _TraceMode: TypeAlias = Literal["array", "read", "write", "unset"]
 
 class Variable:
-    def __init__(self, master: Misc | None = ..., value: Any | None = ..., name: str | None = ...) -> None: ...
-    def set(self, value: Any) -> None: ...
+    def __init__(self, master: Misc | None = ..., value: Incomplete | None = ..., name: str | None = ...) -> None: ...
+    def set(self, value: Incomplete) -> None: ...
```
Comment (Akuli) [inline stdlib/tkinter/__init__.pyi:282]:
```suggestion
    def set(self, value) -> None: ...
```

File: stdlib/tkinter/__init__.pyi [hunk 18]
```diff
@@ -277,11 +278,11 @@ def NoDefaultRoot() -> None: ...
...
     initialize = set
-    def get(self) -> Any: ...
-    def trace_add(self, mode: _TraceMode, callback: Callable[[str, str, str], Any]) -> str: ...
+    def get(self) -> Incomplete: ...
```
Comment (Akuli) [inline stdlib/tkinter/__init__.pyi:284]:
```suggestion
    def get(self): ...
```

File: stdlib/tkinter/__init__.pyi (hunk 15 above)
Comment (Akuli) [reply stdlib/tkinter/__init__.pyi:281]: Indeed. I deleted `Incomplete`s like this manually, but apparently missed a couple.

**Review by srittau — APPROVED** [review]
Thanks, this is very useful! I skimmed it, and while there are a few more places where the `Incomplete` could be removed, since this is only a stylistic nit, we can merge as is.

File: stdlib/tkinter/__init__.pyi [hunk 19]
```diff
@@ -1126,7 +1131,7 @@ class Canvas(Widget, XView, YView):
         insertontime: int = ...,
         insertwidth: _ScreenUnits = ...,
         name: str = ...,
-        offset: Any = ...,  # undocumented
+        offset: Incomplete = ...,  # undocumented
```
Comment (AlexWaygood) [inline stdlib/tkinter/__init__.pyi:1134]:
```suggestion
        offset=...,  # undocumented
```

File: stdlib/tkinter/__init__.pyi [hunk 20]
```diff
@@ -1165,7 +1170,7 @@ class Canvas(Widget, XView, YView):
         insertofftime: int = ...,
         insertontime: int = ...,
         insertwidth: _ScreenUnits = ...,
-        offset: Any = ...,  # undocumented
+        offset: Incomplete = ...,  # undocumented
```
Comment (AlexWaygood) [inline stdlib/tkinter/__init__.pyi:1173]:
```suggestion
        offset=...,  # undocumented
```

File: stdlib/tkinter/filedialog.pyi [hunk 21]
```diff
@@ -41,21 +41,21 @@ class FileDialog:
...
     cancel_button: Button
     def __init__(
-        self, master, title: Any | None = ...
+        self, master, title: Incomplete | None = ...
     ) -> None: ...  # title is usually a str or None, but e.g. int doesn't raise en exception either
-    how: Any | None
-    def go(self, dir_or_file: Any = ..., pattern: str = ..., default: str = ..., key: Any | None = ...): ...
-    def quit(self, how: Any | None = ...) -> None: ...
+    how: Incomplete | None
+    def go(self, dir_or_file: Incomplete = ..., pattern: str = ..., default: str = ..., key: Incomplete | None = ...): ...
```
Comment (AlexWaygood) [inline stdlib/tkinter/filedialog.pyi:47]:
```suggestion
    def go(self, dir_or_file=..., pattern: str = ..., default: str = ..., key: Incomplete | None = ...): ...
```

File: stdlib/tkinter/ttk.pyi [hunk 22]
```diff
@@ -77,7 +78,7 @@ class Button(Widget):
         default: Literal["normal", "active", "disabled"] = ...,
         image: tkinter._ImageSpec = ...,
         name: str = ...,
-        padding: Any = ...,  # undocumented
+        padding: Incomplete = ...,  # undocumented
```
Comment (AlexWaygood) [inline stdlib/tkinter/ttk.pyi:81]:
```suggestion
        padding=...,  # undocumented
```

File: stdlib/tkinter/ttk.pyi [hunk 23]
```diff
@@ -96,7 +97,7 @@ class Button(Widget):
         cursor: tkinter._Cursor = ...,
         default: Literal["normal", "active", "disabled"] = ...,
         image: tkinter._ImageSpec = ...,
-        padding: Any = ...,
+        padding: Incomplete = ...,
```
Comment (AlexWaygood) [inline stdlib/tkinter/ttk.pyi:100]:
```suggestion
        padding=...,
```

File: stdlib/tkinter/ttk.pyi [hunk 24]
```diff
@@ -123,7 +124,7 @@ class Checkbutton(Widget):
         name: str = ...,
         offvalue: Any = ...,
         onvalue: Any = ...,
-        padding: Any = ...,  # undocumented
+        padding: Incomplete = ...,  # undocumented
```
Comment (AlexWaygood) [inline stdlib/tkinter/ttk.pyi:127]:
```suggestion
        padding=...,  # undocumented
```

File: stdlib/tkinter/ttk.pyi [hunk 25]
```diff
@@ -147,7 +148,7 @@ class Checkbutton(Widget):
         image: tkinter._ImageSpec = ...,
         offvalue: Any = ...,
         onvalue: Any = ...,
-        padding: Any = ...,
+        padding: Incomplete = ...,
```
Comment (AlexWaygood) [inline stdlib/tkinter/ttk.pyi:151]:
```suggestion
        padding=...,
```

File: stdlib/tkinter/ttk.pyi [hunk 26]
```diff
@@ -255,8 +256,8 @@ class Combobox(Entry):
         invalidcommand: tkinter._EntryValidateCommand = ...,  # undocumented
         justify: Literal["left", "center", "right"] = ...,
         name: str = ...,
-        postcommand: Callable[[], Any] | str = ...,
-        show: Any = ...,  # undocumented
+        postcommand: Callable[[], object] | str = ...,
+        show: Incomplete = ...,  # undocumented
```
Comment (AlexWaygood) [inline stdlib/tkinter/ttk.pyi:260]:
```suggestion
        show=...,  # undocumented
```

File: stdlib/tkinter/ttk.pyi [hunk 27]
```diff
@@ -280,8 +281,8 @@ class Combobox(Entry):
         height: int = ...,
         invalidcommand: tkinter._EntryValidateCommand = ...,
         justify: Literal["left", "center", "right"] = ...,
-        postcommand: Callable[[], Any] | str = ...,
-        show: Any = ...,
+        postcommand: Callable[[], object] | str = ...,
+        show: Incomplete = ...,
```
Comment (AlexWaygood) [inline stdlib/tkinter/ttk.pyi:285]:
```suggestion
        show=...,
```

File: stdlib/tkinter/ttk.pyi [hunk 28]
```diff
@@ -308,8 +309,8 @@ class Combobox(Entry):
...
```
Comment (AlexWaygood) [inline stdlib/tkinter/ttk.pyi:313]:
```suggestion
        show=...,
```

File: stdlib/tkinter/ttk.pyi [hunk 29]
```diff
@@ -477,7 +478,7 @@ class Menubutton(Widget):
         image: tkinter._ImageSpec = ...,
         menu: tkinter.Menu = ...,
         name: str = ...,
-        padding: Any = ...,  # undocumented
+        padding: Incomplete = ...,  # undocumented
```
Comment (AlexWaygood) [inline stdlib/tkinter/ttk.pyi:481]:
```suggestion
        padding=...,  # undocumented
```

File: stdlib/tkinter/ttk.pyi [hunk 30]
```diff
@@ -496,7 +497,7 @@ class Menubutton(Widget):
         direction: Literal["above", "below", "left", "right", "flush"] = ...,
         image: tkinter._ImageSpec = ...,
         menu: tkinter.Menu = ...,
-        padding: Any = ...,
+        padding: Incomplete = ...,
```
Comment (AlexWaygood) [inline stdlib/tkinter/ttk.pyi:500]:
```suggestion
        padding=...,
```

File: stdlib/tkinter/ttk.pyi [hunk 31]
```diff
@@ -546,7 +547,7 @@ class Notebook(Widget):
         sticky: str = ...,  # consists of letters 'n', 's', 'w', 'e', no repeats, may be empty
         padding: tkinter._Padding = ...,
         text: str = ...,
-        image: Any = ...,  # Sequence of an image name, followed by zero or more (sequences of one or more state names followed by an image name)
+        image: Incomplete = ...,  # Sequence of an image name, followed by zero or more (sequences of one or more state names followed by an image name)
```
Comment (AlexWaygood) [inline stdlib/tkinter/ttk.pyi:550]:
```suggestion
        image=...,  # Sequence of an image name, followed by zero or more (sequences of one or more state names followed by an image name)
```

File: stdlib/tkinter/ttk.pyi [hunk 32]
```diff
@@ -662,7 +663,7 @@ class Radiobutton(Widget):
         cursor: tkinter._Cursor = ...,
         image: tkinter._ImageSpec = ...,
         name: str = ...,
-        padding: Any = ...,  # undocumented
+        padding: Incomplete = ...,  # undocumented
```
Comment (AlexWaygood) [inline stdlib/tkinter/ttk.pyi:666]:
```suggestion
        padding=...,  # undocumented
```

File: stdlib/tkinter/ttk.pyi [hunk 33]
```diff
@@ -682,7 +683,7 @@ class Radiobutton(Widget):
         compound: _TtkCompound = ...,
         cursor: tkinter._Cursor = ...,
         image: tkinter._ImageSpec = ...,
-        padding: Any = ...,
+        padding: Incomplete = ...,
```
Comment (AlexWaygood) [inline stdlib/tkinter/ttk.pyi:686]:
```suggestion
        padding=...,
```

File: stdlib/tkinter/ttk.pyi [hunk 34]
```diff
@@ -869,7 +870,7 @@ class Spinbox(Entry):
         invalidcommand: tkinter._EntryValidateCommand = ...,  # undocumented
         justify: Literal["left", "center", "right"] = ...,  # undocumented
         name: str = ...,
-        show: Any = ...,  # undocumented
+        show: Incomplete = ...,  # undocumented
```
Comment (AlexWaygood) [inline stdlib/tkinter/ttk.pyi:873]:
```suggestion
        show=...,  # undocumented
```

File: stdlib/tkinter/ttk.pyi [hunk 35]
```diff
@@ -898,7 +899,7 @@ class Spinbox(Entry):
         increment: float = ...,
         invalidcommand: tkinter._EntryValidateCommand = ...,
         justify: Literal["left", "center", "right"] = ...,
-        show: Any = ...,
+        show: Incomplete = ...,
```
Comment (AlexWaygood) [inline stdlib/tkinter/ttk.pyi:902]:
```suggestion
        show=...,
```

**Comment by Akuli** [comment]
Don't know how I missed all of those :D I'll make a new PR.

**Pull request merged**